webhook/client.go

```diff
@@ -227,7 +231,22 @@ func constructEvent(payload []byte, sigHeader string, secret string, options Con
 	}
 
 	return e, nil
+}
+
+func checkEventNotification(payload []byte) error {
+	e := struct {
+		APIVersion string `json:"api_version"`
+	}{}
+	if err := json.Unmarshal(payload, &e); err != nil {
+		return fmt.Errorf("Failed to parse webhook body json: %s", err.Error())
+	}
+
+	// Snapshot events have an API version field, but event notifications do not
+	if e.APIVersion == "" {
```

Comment:
the `object` field is the canonical way to tell objects apart from one another. So while it's true `api_version` happens to work for v1 event vs v2 event today, the check should really be `object != "event"`